blueocean-personalization/src/main/js/components/StopPropagation.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Why is this a component, rather than a single line inside event handlers that need it?